docs/modules/ROOT/examples/example-app-gitea.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: gitea
+  labels:
+    app.kubernetes.io/name: demo-app
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: demo-app
+  replicas: 1
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: demo-app
+    spec:
+      containers:
+        - name: demoapp
+          image: quay.io/gpte-devops-automation/gitea:1.19.0
```

Comment:
I usually use `latest` and not a pinned version in these examples. The "What’s next?" talks about it then...